frontend/app/views/fragments/form/submit.scala.html

```diff
@@ -44,3 +46,16 @@
         </div>
     </div>
 </fieldset>
+
+@countryGroup.map{ cg =>
+    @if(cg.id.toLowerCase == "us") {
+        <div class="us-legal">
+            NOTICE REGARDING AUTOMATIC RENEWAL: IF YOU DO NOT CANCEL YOUR MEMBERSHIP PRIOR TO THE END OF YOUR INITIAL
```

Comment:
It's a shame this has to be in capitals. Did we query back to double check this is absolutely required?